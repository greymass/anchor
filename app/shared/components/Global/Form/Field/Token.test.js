import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import GlobalFormFieldToken from './Token';

const mockProps = {
  icon: 'icon',
  label: 'multi token field',
  loading: false,
  name: 'multi token field',
  placeholder: 'string placeholder',
};

describe('GlobalFormFieldToken', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<GlobalFormFieldToken {...mockProps} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
