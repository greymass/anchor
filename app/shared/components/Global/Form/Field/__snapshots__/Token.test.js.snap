// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalFormFieldToken renders correctly 1`] = `
<FormField
  control={[Function]}
  icon="icon"
  label="multi token field"
  loading={false}
  name="multi token field"
  onChange={[Function]}
  placeholder="string placeholder"
/>
`;
